app/src/main/java/com/sun/mangareader01/ui/adapter/TrendingPagerAdapter.kt

```diff
@@ -0,0 +1,75 @@
+package com.sun.mangareader01.ui.adapter
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+import com.sun.mangareader01.R
+import com.sun.mangareader01.data.model.Manga
+import com.sun.mangareader01.ui.listener.OnItemClickListener
+import kotlinx.android.synthetic.main.fragment_trending_pager.layoutRefresh
+import kotlinx.android.synthetic.main.fragment_trending_pager.recyclerMangas
+
+class TrendingPagerAdapter(
+    fragmentManager: FragmentManager,
+    private val fragments: List<PagerFragment>
+) : FragmentStatePagerAdapter(
+    fragmentManager,
+    BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT
+) {
+
+    override fun getItem(position: Int) = fragments[position]
+
+    override fun getCount() = fragments.size
+
+    override fun getPageTitle(position: Int) = getItem(position).title
+
+    class PagerFragment(
```

Comment:
không nên có params trong fragment vì khi xoay thiết bị nó sẽ overloading lại với non-params. thay vào đó tạo ra file newInstance()
https://stackoverflow.com/questions/9245408/best-practice-for-instantiating-a-new-android-fragment